client/src/pages/login/applicantDash/applicant.css

```diff
@@ -0,0 +1,12 @@
+.applicant{
+   	color: green;   /* sets big word to green */
```

Comment:
Extra tab here! Linters are your friend.

Also, I notice a lot of these .CSS files are almost identical (```applicant.css```,```volunteer.css```,```signup.css```,etc.) Is there a way those sorts of CSS blocks could be consolidated?

Reply:
Yes, those are all similar atm, but I structured the format in a manner where multiple `.jsx` files are not dependent on a main `.css` file. There are respective `.jsx` files for each `.css` because down the line I set it up for us to just swap them out with capable pages. Those pages are just place holders. Less overhead when we start replacing those files, in my opinion. Also easier to debug.

Reply:
That makes sense. Code review is just a good opportunity to make sure these sorts of things are being considered by the developer. Shouldn't just be a list of things to fix in the code!